exercises/error-handling/error_handling_test.go

```diff
@@ -27,6 +27,14 @@ func (mr mockResource) Close() error      { return mr.close() }
 func (mr mockResource) Frob(input string) { mr.frob(input) }
 func (mr mockResource) Defrob(tag string) { mr.defrob(tag) }
 
+// If this test fails and you've properly defined testVersion the requirements
```

Comment:
Probably not, but since the previous author added it and its not wrong I left it. Removing those comments can go along with #470 and #515. hello-world should be the definitive documentation on how the test versioning works for all the tests.